src/main.rs

```diff
@@ -122,7 +138,10 @@ fn load_and_index_pois(mut rubber: Rubber, conn: &Connection, dataset: &str) {
     let rows = stmt.lazy_query(&trans, &[], PG_BATCH_SIZE).unwrap();
 
     let pois = rows.iterator()
-        .map(|r| build_poi(r.unwrap(), &admins_geofinder));
+        .filter_map(|r| r.ok())
```

Comment:
I would be nice to have a log on error here.

you can do a `map_err` like I've done above

Reply:
Done.